Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3688,6 +3688,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         case CSSPropertyWebkitTextZoom:
         case CSSPropertyAdditiveSymbols:
         case CSSPropertyAlignmentBaseline:
+        case CSSPropertyAlignTracks:
```

Comment:
Please file a bug to add animation support for this property and reference it here with a `FIXME`.

Reply:
That said, I expect this single line would do:

`new DiscretePropertyWrapper<AlignTracks>(CSSPropertyAlignTracks, &RenderStyle::alignTracks, &RenderStyle::setAlignTracks)`

… and make sure there is WPT coverage for this, if it doesn't exist.

Reply:
I did try adding that line, and was running into compile errors on line 3477 was running into some compile issues. Will need to look into that more.

Reply:
@stwrt Does `new DiscretePropertyWrapper<const Vector<StyleContentAlignmentData>&>(...)` work?

Reply:
Still seeing build errors

Reply:
@stwrt which ones are you seeing?

Reply:
I'm still curious what the failures are.